packages/airnode-deployer/src/utils/validation.ts

```diff
@@ -2,22 +2,31 @@ import { Config } from '@api3/airnode-node';
 import { validateJsonWithTemplate } from '@api3/airnode-validator';
 import * as logger from '../utils/logger';
 
-export function validateConfig(config: Config, nodeVersion: string, skipVersionCheck: boolean) {
-  if (!skipVersionCheck && nodeVersion !== config.nodeSettings.nodeVersion) {
-    logger.fail(
-      `nodeVersion under nodeSettings in config.json is ${config.nodeSettings.nodeVersion} while the deployer node version is ${nodeVersion}`
-    );
-    throw new Error("Node version specified in config.json does not match the deployer's version");
+export function validateConfig(config: Config, nodeVersion: string) {
+  if (config.nodeSettings.skipValidation) {
+    logger.warn('Skipping the config validation');
+    return;
   }
 
   if (config.nodeSettings.cloudProvider.type === 'local') {
     const message = "Deployer can't deploy to 'local' cloud provider";
     logger.fail(message);
     throw new Error(message);
   }
+
+  const result = validateJsonWithTemplate(config, `config@${nodeVersion}`);
+
+  if (!result.valid) {
```

Comment:
shouldn't this if sentence follow the same logic as in [node](https://github.com/api3dao/airnode/blob/b79b5f603ad49c4ff3b1a3a5f2b3389e5c85746b/packages/airnode-node/src/config/index.ts#L30) package?

Reply:
It does, its only split into 2 conditions so that warning is issued in deployer informing user about skipped validation step